src/main/java/org/openjdk/jextract/impl/HeaderFileBuilder.java

```diff
@@ -173,7 +161,7 @@ class Holder {
                 appendBlankLine();
                 emitDocComment(decl);
                 appendLines(STR."""
-                public static \{retType} \{javaName}(\{paramExprs(declType, finalParamNames, isVarArg)}) {
+                \{"public static"} \{retType} \{javaName}(\{paramExprs(declType, finalParamNames, isVarArg)}) {
```

Comment:
I suppose this should just be:

```suggestion
                public static \{retType} \{javaName}(\{paramExprs(declType, finalParamNames, isVarArg)}) {
```